lib/pbio/drv/legodev/legodev_test.c

```diff
@@ -46,6 +81,9 @@ void pbdrv_legodev_init(void) {
             legodev->uart_dev = pbdrv_legodev_pup_uart_configure(0, 0, dcmotor);
         }
     }
+
+    // We would normally start the process here, but this also causes it to
+    // start in other tests where this is not needed.
```

Comment:
So perhaps we should have a `#if !...TEST...` here and start the process here in normal firmware builds?

Reply:
The issue isn't in test vs. normal builds, but between test_x vs. test_y.